### PR TITLE
add comments to oomstore

### PR DESCRIPTION
**Pull request by YikSanChan**

### CODE REVIEW

File: pkg/oomstore/oomstore.go [hunk 1]
```diff
@@ -47,13 +53,15 @@ func Open(ctx context.Context, opt types.OomStoreConfig) (*OomStore, error) {
 	}, nil
 }
 
+// Create necessary tables in the Metadata Store to manage metadata for OomStore.
 func Create(ctx context.Context, opt types.OomStoreConfig) (*OomStore, error) {
```
Comment (jinghancc) [inline pkg/oomstore/oomstore.go:57]: add comments: use `Create` for creating the new OomStore instance

**Review by lianxmfor — APPROVED** [review]
lgtm

File: pkg/oomstore/sync.go [hunk 2]
```diff
@@ -11,7 +11,10 @@ import (
 	"github.com/oom-ai/oomstore/pkg/oomstore/types"
 )
 
-// Move a certain feature group revision data from offline to online store
+// Move a particular revision of a feature group from offline to online store.
```
Comment (jinghancc) [inline pkg/oomstore/sync.go:14]: Move -> Sync

Comment (wfxr) [inline pkg/oomstore/sync.go:14]: Suggestion: replace `Move` with `Sync` or `Copy`

**Review by jinghancc — APPROVED** [review]
lgtm

**Review by wfxr — APPROVED** [review]
LGTM

**Pull request merged**